src/components/date/date-test.stories.tsx

```diff
@@ -148,3 +150,55 @@ export const MultipleDates: StoryObj<typeof DateInput> = () => {
 MultipleDates.storyName =
   "Multiple Dates with onPickerOpen and onPickerClose callbacks";
 MultipleDates.parameters = { chromatic: { disableSnapshot: true } };
+
+interface I18nArgs extends CommonTextboxArgs {
+  /** An Accordion or list of Accordion components to be rendered inside the AccordionGroup */
```

Comment:
**question**: the description of this prop doesn't seem to match what its used for, is this intentional?